hack/install-cli.sh

```diff
@@ -237,6 +237,15 @@ setup_binary() {
   info "Installing ${INSTALL_CLI_TYPE} to ${INSTALL_LOCATION}/${INSTALL_CLI_TYPE}"
   tar -xzof "${TMP_BIN}" -C "${TMP_DIR}" --no-same-owner
 
+  # in some production environments, directory /usr/local/bin may not exist
```

Comment:
This comment is probably inappropriate, what we should do is `ensure the path of $INSTALL_LOCATION exist`, this is a configurable environment and not limited to `/usr/local/bin`.